crates/pera-core/src/dwallet_mpc/mpc_outputs_verifier.rs

```diff
@@ -58,6 +63,24 @@ pub struct OutputVerificationResult {
 
 impl DWalletMPCOutputsVerifier {
     pub fn new(epoch_store: &AuthorityPerEpochStore) -> Self {
+        let weighted_parties: HashMap<PartyID, PartyID> = epoch_store
```

Comment:
```suggestion
        let weighted_parties = epoch_store
```
no need for the type, it's already in the collect().